contracts/Escrow.sol

```diff
@@ -520,46 +585,25 @@ contract Escrow is IEscrow {
         unstETHIds = _batchesQueue.getNextWithdrawalsBatches(limit);
     }
 
-    /// @notice Returns whether all withdrawal batches have been closed.
-    /// @return isWithdrawalsBatchesClosed A boolean value indicating whether all withdrawal batches have been
-    ///     closed (`true`) or not (`false`).
-    function isWithdrawalsBatchesClosed() external view returns (bool) {
+    /// @notice Retrieves details about the current state of the rage quit escrow.
+    /// @return details A `RageQuitEscrowDetails` struct containing the following fields:
+    /// - `isWithdrawalsBatchesClosed`: Indicates whether the withdrawals batches are closed.
```

Comment:
Move `isWitdrawalsBatchesClosed` and `isRageQuitExtensionPeriodStarted` into standalone getters